frontend/packages/shared/src/hooks/mutations/useCreateTextResourcesForOrgMutation.ts

```diff
@@ -0,0 +1,13 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useServicesContext } from 'app-shared/contexts/ServicesContext';
+import { QueryKey } from 'app-shared/types/QueryKey';
+
+export const useCreateTextResourcesForOrgMutation = (org: string, language: string) => {
+  const q = useQueryClient();
+  const { createTextResourcesForOrg } = useServicesContext();
+  return useMutation({
+    mutationFn: () => createTextResourcesForOrg(org, language),
+    onSuccess: () =>
+      Promise.all([q.invalidateQueries({ queryKey: [QueryKey.TextResourcesForOrg] })]),
```

Comment:
`Promise.all` er vel ikke nødvendig når det bare er én `Promise`? Her ville jeg for øvrig prøvd å unngå `invalidateQueries` fullstendig for å unngå unødvendige nettverkskall. Har du vurdert å oppdatere dataen med [setQueryData](https://tanstack.com/query/v5/docs/reference/QueryClient/#queryclientsetquerydata)?

Reply:
Det er et veldig godt poeng, usikker hvorfor det ble sånn

Reply:
Har du vurdert å oppdatere dataen med [setQueryData](https://tanstack.com/query/v5/docs/reference/QueryClient/#queryclientsetquerydata)?